packages/trader/src/Modules/MT5/Containers/mt5-compare-accounts-modal.jsx

```diff
@@ -46,140 +46,154 @@ const MT5AttributeDescriber = ({ name, tooltip, counter }) => {
-            financial_stp: localize('FX-majors, FX-minors, FX-exotics, Cryptocurrencies'),
-        },
-    ]);
+const compareAccountsData = ({ landing_companies, is_eu }) =>
+    filterAvailableAccounts(
+        landing_companies,
+        [
+            {
+                attribute: <MT5AttributeDescriber name={localize('Account currency')} />,
+                synthetic: localize('USD'),
+                financial: localize('USD'),
+                financial_stp: localize('USD'),
+            },
+            {
+                attribute: (
+                    <MT5AttributeDescriber
+                        name={localize('Maximum leverage')}
+                        counter={1}
+                        tooltip={localize(
+                            'Leverage gives you the ability to trade a larger position using your existing capital. Leverage varies across different symbols.'
+                        )}
+                    />
+                ),
+                synthetic: localize('Up to 1:1000'),
+                financial: localize('Up to 1:1000'),
+                financial_stp: localize('Up to 1:100'),
+            },
+            {
+                attribute: (
+                    <MT5AttributeDescriber
+                        name={localize('Order execution')}
+                        counter={2}
+                        tooltip={localize(
+                            "All 3 account types use market execution. This means you agree with the broker's price in advance and will place orders at the broker's price."
+                        )}
+                    />
+                ),
+                synthetic: localize('Market'),
+                financial: localize('Market'),
+                financial_stp: localize('Market'),
+            },
+            {
+                attribute: (
+                    <MT5AttributeDescriber
+                        name={localize('Spread')}
+                        counter={3}
+                        tooltip={localize(
+                            "The spread is the difference between the buy price and sell price. A variable spread means that the spread is constantly changing, depending on market conditions. A fixed spread remains constant but is subject to alteration, at the Broker's absolute discretion."
+                        )}
+                    />
+                ),
+                synthetic: localize('Fixed/Variable'),
+                financial: localize('Variable'),
+                financial_stp: localize('Variable'),
+            },
+            {
+                attribute: (
+                    <MT5AttributeDescriber
+                        name={localize('Commission')}
+                        counter={4}
+                        tooltip={localize(
+                            'Deriv charges no commission across all account types, except cryptocurrency accounts.'
```

Comment:
we had text changes in the latest PR, can you fetch the changes?

Reply:
Done.